src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.Android.cs

```diff
@@ -53,25 +54,29 @@ private async Task<ImageData> GetSvgImageDataAsync(CancellationToken ct)
 				return await ReadFromStreamAsync(fileStream, ct);
 			}
 
-			if (AbsoluteUri != null)
+			if (AbsoluteUri is not null)
 			{
 				// The ContactsService returns the contact uri for compatibility with UniversalImageLoader - in order to obtain the corresponding photo we resolve using the service below.
 				if (IsContactUri(AbsoluteUri))
 				{
-					var stream = ContactsContract.Contacts.OpenContactPhotoInputStream(ContextHelper.Current.ContentResolver, Android.Net.Uri.Parse(AbsoluteUri.OriginalString));
+					if (ContactsContract.Contacts.OpenContactPhotoInputStream(ContextHelper.Current.ContentResolver, Android.Net.Uri.Parse(AbsoluteUri.OriginalString)) is not { } stream)
+					{
+						return ImageData.Empty;
+					}
+
 					return await ReadFromStreamAsync(stream, ct);
 				}
 
 				if (Downloader is not null)
 				{
 					var filePath = await Download(ct, AbsoluteUri);
 
-					if (filePath == null)
+					if (filePath is null)
 					{
 						return ImageData.Empty;
 					}
 
-					using var fileStream = File.OpenRead(FilePath);
+					using var fileStream = File.OpenRead(filePath.LocalPath);
```

Comment:
@MartinZikmund I updated this, but I wanted to validate it with you.

Reply:
It's likely that it was a bug for this portion, and it's not been used in a while. Good catch!